### ISSUE TITLE
Cannot build Markdown or Open XML

### ISSUE THREAD

**Issue by bonimy** [EWSoftware/SHFB#442]
tl;dr to the code block.

These observations were done on a fresh install of Genuine Windows 10 Home Edition.

I did an install of [Visual Studio Community 2017 (15.0.26228.9)](https://www.visualstudio.com/downloads/) to "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community" (Same drive path as my OS).

I installed all workloads and the following Individual Components:
- All .NET
- All Cloud, database, and server
- All Code tools
- All Compilers, build tools, and runtimes
- All Debugging and testing
- All Development activities
- All Games and Graphics
- All SDKs, libraries, and frameworks **except**:
    - All Android NDKs except Android NDK (R13B)
    - TypeScript 2.0 SDK

I also installed the following extensions from the Visual Studio Marketplace:
- [Extensibility Tools](https://github.com/madskristensen/ExtensibilityTools)
- [GitHub Extension for Visual Studio](https://visualstudio.github.com/)
- [Microsoft Visual Studio 2017 Installer Projects](https://marketplace.visualstudio.com/items?itemName=VisualStudioProductTeam.MicrosoftVisualStudio2017InstallerProjects)
- [Markdown Editor](https://github.com/EWSoftware/MarkdownEditor)

I then ran the [SHFB Installer v20171.28.0](https://github.com/EWSoftware/SHFB/releases) which installed the HTML Help 1 Compiler, Sandcastle Help File Builder and Tools, VSPackage, Schemas, and Snippets. I later installed the Spell checker through the marketplace. I created a SHFB project through the SHFB GUI for my project, selected Markdown (md) as the sole build, and got this output when building:

```
-------------------------------
[Sandcastle Help File Builder and Tools, version 2017.1.28.0]
Creating output and working folders...
Finding tools...
The Sandcastle tools are located in 'C:\Program Files (x86)\EWSoftware\Sandcastle Help File Builder\'
Locating components in the following folder(s):
    C:\Users\nrgarcia\Desktop
    C:\ProgramData\EWSoftware\Sandcastle Help File Builder\Components and Plug-Ins\
    C:\Program Files (x86)\EWSoftware\Sandcastle Help File Builder\
Using framework reflection data for '.NET Framework 4.5' located in 'C:\Program Files (x86)\EWSoftware\Sandcastle Help File Builder\Data\.NETFramework'
Using presentation style 'VS2013' located in 'C:\Program Files (x86)\EWSoftware\Sandcastle Help File Builder\PresentationStyles\VS2013'
    Last step completed in 00:00:00.5080
-------------------------------

SHFB: Error BE0074: The selected presentation style (VS2013) does not support one or more of the selected help file formats.  Supported formats: HtmlHelp1, MSHelpViewer, Website
   at SandcastleBuilder.Utils.BuildEngine.BuildProcess.Build()
```
I've tried looking online, and through the issues of this project, but got no useful information. I got the same error when trying to build Open XML. All other formats built fine. I would like to compile a Markdown Wiki for GitHub.

Thanks

**Comment by EWSoftware** [comment]
> The selected presentation style (VS2013) does not support one or more of the selected
> help file formats.  Supported formats: HtmlHelp1, MSHelpViewer, Website

As noted in the error message, the selected presentation style does not support the selected help file format.  Markdown and Open XML generate completely different file formats that aren't compatible with the HTML-based presentation styles like VS2013.

You can't combine those two help file formats with any other formats and they must use a matching presentation style.  You need to select the help file format appropriate to the presentation style (Markdown for Markdown and Open XML for Open XML) and build each individually.

The help file format is selected in the [Build category](http://ewsoftware.github.io/SHFB/html/da405a33-3eeb-4451-9aa8-a55be5026434.htm#HelpFileFormats) of project properties.  The presentation style is selected in the [Help File category](http://ewsoftware.github.io/SHFB/html/1b2dff59-92cc-4578-b261-f3849f30c26c.htm#PresentationStyle).

**Comment by bonimy** [comment]
@EWSoftware, thank you for the response. That fixed the issue.

Perhaps it would be helpful to specify in the program that one option relies on the other. If Markdown and Open XML need their own presentation styles (as a new user, I didn't understand these topics, even after a cursory glance of the SHFB documentation), then it might help to modify the GUI to let the user know that this presentation style is necessary, or to automate the process.

**Comment by EWSoftware** [comment]
Okay, I'll see what I can do to make it more apparent.
